torch/csrc/jit/passes/quantization.cpp

```diff
@@ -481,16 +490,30 @@ class InsertObserversHelper {
       const script::Module& module,
       const std::string& method_name);
 
+  // Fill the map from values to the list of values that can pass the observed
+  // property to it
+  void fillPassThroughValueMap(const std::shared_ptr<Graph>& graph);
+
   const ModuleQConfigMap& module_qconfig_map_;
   // Values we want to skip observing, used to skip values in
   // the middle of the ops that are supposed to be fused, e.g.
   // the output value of conv in the conv - relu pattern
   std::unordered_set<Value*> values_to_skip_;
   std::unordered_set<Graph*> visited_graph_of_observer_map_;
   std::unordered_map<Value*, script::Module> observer_for_value_;
+  std::unordered_map<Value*, Value*> caller_to_callee_;
```

Comment:
Is this comment correct? This seems like a description of observer_for_value

Reply:
which comment?